packages/block-editor/src/store/reducer.js

```diff
@@ -1470,6 +1472,31 @@ export function highlightedBlock( state, action ) {
 	return state;
 }
 
+/**
+ * Reducer returning the blocks whose inner blocks are controlled. Maps a block
+ * clientId to a boolean. If true, the block's inner blocks are controlled by
+ * the block itself. For example, a template part uses an entity provider to
+ * control its blocks.
+ *
+ * @param {boolean} state  Current blocks which have controlled inner blocks.
+ * @param {Object}  action Dispatched action.
+ *
+ * @return {string} Updated state.
+ */
+export function controlledInnerBlocks(
```

Comment:
I believe this should be part of the `blocks` reducer (sub reducer there)

Reply:
👍